linera-core/src/data_types.rs

```diff
@@ -313,7 +313,7 @@ impl<T> ClientOutcome<T> {
         }
     }
 
-    #[expect(clippy::result_large_err)]
+    #[allow(clippy::result_large_err)]
```

Comment:
(So whether a `ClientError` is too large now depends on the selected features?)

Reply:
AFAIU I think it's because I changed the `BlobsNotFound` error to be a vector, instead of always having one `BlobId`. So technically now it would cross the threshold of being large or not depending on the vector size.

Reply:
I think this is only about the size on the stack. The vector's _entries_ are on the heap.